packages/client/libclient-py/quickmpc/__init__.py

```diff
@@ -11,4 +12,6 @@
     "Schema",
     "ShareValueTypeEnum",
     "__version__",
+    "parse",
+    "parse_csv"
```

Comment:
I thought it would be better not to release features that users do not use.
I thought the test should be modified since it seems to be used only for testing.

PS
However, there are other things I wanted to fix in the test, and I thought it would be better to do it all together, so I don't need to fix them here.